cmd/localstack/hotreloading.go

```diff
@@ -16,8 +16,8 @@ type ChangeListener struct {
 	watchedFolders     []string
 }
 
-func NewChangeListener(debouncingInterval time.Duration) (*ChangeListener, error) {
-	watcher, err := filenotify.New(200 * time.Millisecond)
+func NewChangeListener(debouncingInterval time.Duration, fileWatcher string) (*ChangeListener, error) {
+	watcher, err := filenotify.New(200*time.Millisecond, fileWatcher)
```

Comment:
Should we maybe make the interval configurable as well while we're at it?

Reply:
I am kind of conflicted. On the one hand, it might make sense to do, if they want to increase it when running huge folders. On the other hand, I am not sure how much it will really impact.